app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    try:
+        for friend in friends:
+            try:
+                cafe.visit_cafe(friend)
+            except NotWearingMaskError:
+                masks_to_buy += 1
+        if masks_to_buy > 0:
+            return f"Friends should buy {masks_to_buy} masks"
+    except VaccineError:
+        return "All friends should be vaccinated"
```

Comment:
Try to avoid nesting `try except` blocks, use one `try except` block with multiple `except` statements instead